### ISSUE TITLE
Improvements to 2D meshing workflow

### ISSUE THREAD

**Issue by stefsmeets** [hpgem/nanomesh#32]
This PR tracks ideas for follow-up to #26 

- [ ] Generate periodic 2d mesh
- [x] Crop ROI from `Plane` (#41)
- [x] Add cropping method (select one pore) (#41)
- [x] Generate inside mesh as well as outside mesh in same function (#58)
- [x] Handle segmentation better (instead of `seg = Plane.from_array(1.0 * (plane_gauss.array_view >= thresh))`
    -> Use `seg.apply(np.digitize, bins=[thresh])`
- [ ] Implement periodicity by tiling image with a wrap-around buffer (i.e. tile 3x3 -> triangulate -> and pull out the middle)
- [x] Close contours and use subdivision algorithm to generate edge points
- [ ] Pygalmesh multi-domain for 2d mesh? (i.e. 0==air, 1==Si, 2==?? element)
- [x] Use `trimesh.Trimesh.contains` for point checking

**Comment by stefsmeets** [comment]
Closing this, because after #77 this is in pretty good state. Opened a new issue to keep track of periodic meshing: #81
